docs/src/cookbooks/cookbook.md

```diff
@@ -462,3 +464,52 @@ ChiselStage.emitVerilog(new Salt)
 ```scala mdoc:verilog
 ChiselStage.emitVerilog(new Salt)
 ```
+
+## Directionality
+
+### How do I strip directions from a bidirectional Bundle (or other Data)?
+
+Given a bidirectional port like a `Decoupled`, you will get an error if you try to connect it directly
+to a register:
+
+```scala mdoc:silent
+import chisel3.util.Decoupled
+class BadRegConnect extends Module {
+  val io = IO(new Bundle {
+    val enq = Decoupled(UInt(8.W))
+  })
+  
+  val monitor = RegNext(io.enq)
```

Comment:
```suggestion
  val monitor = Reg(chiselTypeOf(io.enq))
  monitor := io.enq
```

Can you do this without `RegNext`, to make it less different from the thing that works below?

Reply:
sure